app/models/etsource/loader.rb

```diff
@@ -46,8 +46,11 @@ def presets
 
     def area_attributes(area_code)
       @area_attributes[area_code] ||= begin
-        area_attr = Atlas::Dataset.find(area_code).to_hash
+        area = Atlas::Dataset.find(area_code)
+        area_attr = area.to_hash
+        area_attr[:is_derived] = true if area.is_a?(Atlas::Dataset::DerivedDataset)
```

Comment:
First I thought you were right and I made a braindead mistake here, but then I remembered my reason for this: I only want to have this key in the hash if it is true, but not if it is false